publish.js

```diff
@@ -293,8 +293,15 @@ publish.dev = function (isTest, options) {
     // running this module.
     var modulePkg = publish.getPkg(opts.moduleRoot);
 
+    var checkChange = options.checkChanges;
+    if (checkChange === undefined) {
+        checkChange = true;
+    }
```

Comment:
Rather than create the checkChanges variable here and do the if check on undefined, I think you can create a new option in the [defaultOptions](https://github.com/fluid-project/fluid-publish/blob/master/package.json#L41-L55) called something like "allowChanges", and set it to `false` by default. In addition to updating the unit tests, you should update the README to provide documentation for this feature.